src/pages/components/tabs/style.mdx

```diff
@@ -5,6 +5,21 @@ description:
 tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 ---
 
+<PageDescription>
+
+The following page documents visual specifications such as color, typography,
+structure, and size.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Color</AnchorLink> <AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink> <AnchorLink>Size</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
 ## Color
 
 ### Line tabs
```

Comment:
Because you have variants in the color and structure sections, the page gets pretty deep and it can be easy to loose track of where you are. I suggest adding 'color' to the H3 headers:
Line tab color
Dismissible line tab color
Contained tab color
Dismissible contained tab color. 

I suggest doing this in the structure section, too. I.e. Dismissible line tab structure